eleven-group/pickup
# feat: add slot generator

Close #34 

HOW TO TEST :
1. Checkout
2. `make restart`
3. Go inside pickup-php container and run `make install` then `make reset-dev`
4. Check the route /api/shops/:id/slots
